templates/telemetry_derived/error_aggregates/query.sql

```diff
@@ -0,0 +1,196 @@
+-- Get pings from the last day from live tables, stable tables for older
+WITH crash_pings AS (
+  SELECT
+    *
+  FROM
+    telemetry_live.crash_v4
+  WHERE
+    DATE(submission_timestamp) >= DATE_SUB(CURRENT_DATE, INTERVAL 1 DAY)
+  UNION ALL
+  SELECT
+    *
+  FROM
+    telemetry_stable.crash_v4
+  WHERE
+    DATE(submission_timestamp) < DATE_SUB(CURRENT_DATE, INTERVAL 1 DAY)
+),
+main_pings AS (
+  SELECT
+    *
+  FROM
+    telemetry_live.main_v4
+  WHERE
+    DATE(submission_timestamp) >= DATE_SUB(CURRENT_DATE, INTERVAL 1 DAY)
+  UNION ALL
+  SELECT
+    *
+  FROM
+    telemetry_stable.main_v4
+  WHERE
+    DATE(submission_timestamp) < DATE_SUB(CURRENT_DATE, INTERVAL 1 DAY)
+),
+core_pings AS (
+  SELECT
+    *
+  FROM
+    telemetry.core_live
+  WHERE
+    DATE(submission_timestamp) >= DATE_SUB(CURRENT_DATE, INTERVAL 1 DAY)
+  UNION ALL
+  SELECT
+    *
+  FROM
+    telemetry.core
+  WHERE
+    DATE(submission_timestamp) < DATE_SUB(CURRENT_DATE, INTERVAL 1 DAY)
+),
+-- Get main, content, startup, and content_shutdown crashes from crash pings
+crash_ping_data AS (
+  SELECT
+    submission_timestamp,
+    normalized_channel AS channel,
+    environment.build.version,
+    application.display_version,
+    environment.build.build_id,
+    metadata.uri.app_name,
+    IF(metadata.uri.app_name = 'Fennec'
+       AND environment.system.os.name = 'Linux',
+       'Android',
+       environment.system.os.name) AS os,
+    environment.system.os.version AS os_version,
```

Comment:
I think we should be using `normalized_os` and `normalized_os_version` throughout this, unless we have a specific need here. `Fennec` + `Linux` will probably need to stay though, per https://github.com/mozilla/gcp-ingestion/issues/999

Reply:
Sure.  The only "need" is that this requires changing mission control to use normalized values instead

Reply:
the difference should be that `os = 'Darwin'` becomes `normalized_os = 'Mac'`, and unrecognized values are normalized to `Other`.

cc @wlach regarding whether mission control should needs the original values

Reply:
> changing mission control to use normalized values instead

also, we could use `IF(normalized_os = 'Mac', 'Darwin', normalized_os)`.



> the difference should be that `os = 'Darwin'` becomes `normalized_os = 'Mac'`, and unrecognized values are normalized to `Other`.

er, I oversimplified. there's quite a few differences:
- os **contains** `Linux`, `BSD`, or `SunOS` becomes `Linux`
- os **starts with** `Windows` or `WINNT` becomes `Windows`
- os **starts with** `Darwin` becomes `Mac`
- os **starts with** `iOS` or **contains** `iPhone` becomes `iOS`
- os **starts with** `Android` becomes `Android`
- anything else becomes `Other`

and if https://github.com/mozilla/gcp-ingestion/issues/1000 is merged, it will be even more complex.

Reply:
That's a bit of a mess.  It should be easier to use normalized and make the changes in mission control.

Reply:
I'm fine with whatever you want to do here, Ben is actually doing the mission control changes so can presumably adjust things on that side as needed.